UnitTests/GitCommandsTests/Git/RevisionDiffProviderTest.cs

```diff
@@ -20,10 +20,10 @@ public void Setup()
         }
 
 #if !DEBUG
-        //Testcases that should assert in debug; should not occur but undefined behavior that should be blocked in GUI
-        //Cannot compare unstaged to unstaged or staged to staged but give predictive output in release builds
+        // Testcases that should assert in debug; should not occur but undefined behavior that should be blocked in GUI
+        // Cannot compare unstaged to unstaged or staged to staged but give predictive output in release builds
```

Comment:
Actually I see why -- I was building under `DEBUG`, and this code was excluded by the pre-processor. Seems pretty obvious now.